sdk/privatedns/Azure.ResourceManager.PrivateDns/api/Azure.ResourceManager.PrivateDns.netstandard2.0.cs

```diff
@@ -26,7 +26,7 @@ public partial class PrivateDnsAaaaRecordResource : Azure.ResourceManager.ArmRes
         protected PrivateDnsAaaaRecordResource() { }
         public virtual Azure.ResourceManager.PrivateDns.PrivateDnsAaaaRecordData Data { get { throw null; } }
         public virtual bool HasData { get { throw null; } }
-        public static Azure.Core.ResourceIdentifier CreateResourceIdentifier(string subscriptionId, string resourceGroupName, string privateZoneName, string aaaaRecordName) { throw null; }
+        public static Azure.Core.ResourceIdentifier CreateResourceIdentifier(string subscriptionId, string resourceGroupName, string PrivateDnsZoneName, string aaaaRecordName) { throw null; }
```

Comment:
I think here we should use `privateDnsZoneName`, please double check all the related methods.